Source/Shaders/Builtin/Functions/discardIfClippedCombineRegions.glsl

```diff
@@ -0,0 +1,48 @@
+/**
+ * Clip a fragment by an array of clipping planes. Clipping plane regions are combined, so a fragment must be clipped
+ * by all of the planes to be discarded.
+ *
+ * @name czm_discardIfClipped
+ * @glslFunction
+ *
+ * @param {vec4[]} clippingPlanes The array of planes used to clip, defined in eyespace.
+ * @param {int} clippingPlanesLength The number of planes in the array of clipping planes.
+ * @returns {float} The distance away from a clipped fragment, in eyespace
+ */
+float czm_discardIfClippedCombineRegions (vec4[czm_maxClippingPlanes] clippingPlanes, int clippingPlanesLength)
```

Comment:
Throughout, no space before `(` for functions.  WebStorm has an auto formatter: `Ctrl-ALT-L`